protobuf/types/execution.proto

```diff
@@ -109,4 +108,11 @@ message Execution {
     (gogoproto.customtype) = "github.com/mesg-foundation/engine/hash.Hash",
     (gogoproto.nullable) = false
   ];
+
+  // price of running the exeuction.
+  string price = 14 [
+    (gogoproto.moretags) = 'hash:"name:14"',
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (gogoproto.nullable) = false
```

Comment:
```suggestion
```

Reply:
Why you want to remove nullable? sdk.Coins is an array so you can still set null on it.